USE_CASES.md

```diff
@@ -171,6 +172,186 @@ print(response.body)
+
+```
+hello-sendgrid
+├── hello_sendgrid
+│   ├── hello_sendgrid
+│   │   ├── __init__.py
+│   │   ├── settings.py
+│   │   ├── urls.py
+│   │   ├── views.py
+│   │   └── wsgi.py
+│   └── manage.py
+└── requirements.txt
+```
+
+Next we open the file `urls.py` in order to add the view we have just created to the Django URL dispatcher.
+
+```python
+from django.conf.urls import url
+from django.contrib import admin
+
+from .views import index
+
+
+urlpatterns = [
+    url(r'^admin/', admin.site.urls),
+    url(r'^sendgrid/', index, name='sendgrid'),
+]
+```
+
+We also assume that you have set up your development environment with your `SENDGRID_API_KEY`. If you have not done it yet, please do so. See the section [Setup Environment Variables](https://github.com/sendgrid/sendgrid-python#setup-environment-variables).
+
+Now we should be able to send an email. Let's run our Django development server to test it. Find the file `manage.py` then run:
+
+```
+$ python manage.py runserver
+```
+
+By default, it starts the development server at `http://127.0.0.1:8000/`. To test if we can send email or not, go to `http://127.0.0.1:8000/sendgrid/`. If it works, we should see the page says "Email Sent!".
+
+## Deploy to Heroku
+
+Before we start the deployment, let's log in to your Heroku account and create a Heroku app. This tutorial uses `hello-sendgrid`.
+
+We also need to do a couple things:
+
+1. Add `'*'` or your Heroku app domain to `ALLOWED_HOSTS` in the file `settings.py`.
+2. Add `Procfile` with the code below to declare what commands are run by your application's dynos on the Heroku platform.
+
+```
+web: cd hello_sendgrid && gunicorn hello_sendgrid.wsgi --log-file -
+```
+
+The final folder structure looks like this:
+
+```
+hello-sendgrid
+├── hello_sendgrid
+│   ├── hello_sendgrid
+│   │   ├── __init__.py
+│   │   ├── settings.py
+│   │   ├── urls.py
+│   │   ├── views.py
+│   │   └── wsgi.py
+│   └── manage.py
+├── Procfile
+└── requirements.txt
+```
+
+There are different deployment methods we can choose. In this tutorial, we choose to deploy our app using the [Heroku CLI](https://devcenter.heroku.com/articles/heroku-cli). Therefore, let's install it before we go further.
+
+Once you have the Heroku CLI installed, run the command below to log in to your Heroku account if you haven't already.
+
+```
+$ heroku login
+```
+
+Go to the root folder then initialize a Git repository.
+
+```
+$ git init
+$ heroku git:remote -a hello-sendgrid
+```
+
+Since we do not use any static files, we will disable `collectstatic` for this project.
+
+```
+$ heroku config:set DISABLE_COLLECTSTATIC=1
+```
+
+Commit the code to the repository and deploy it to Heroku using Git.
+
+```
+$ git add .
+$ git commit -am "Create simple Hello Email Django app using SendGrid"
+$ git push heroku master
+```
+
+We have not finished yet. We need to go to the Heroku settings and add your `SENDGRID_API_KEY` as one of the Heroku environment variables in the Config Variables section.
```

Comment:
I believe this can be done over the CLI like this: https://devcenter.heroku.com/articles/config-vars#setting-up-config-vars-for-a-deployed-application